### PR TITLE
Remove cloud events as message format for WebHooks, update README.md (#248)

### PR DESCRIPTION
**Pull request by mlagally**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-profile/pull/251.html" title="Last updated on Aug 31, 2022, 1:07 PM UTC (0fd45f3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-profile/251/6ecbc98...0fd45f3.html" title="Last updated on Aug 31, 2022, 1:07 PM UTC (0fd45f3)">Diff</a>

### CODE REVIEW

**Review by benfrancis — APPROVED** [review]

**Comment by mlagally** [comment]
I also updated the readme to reflect the recent name change, see #268

File: README.md [hunk 1]
```diff
@@ -14,14 +14,14 @@ This specification serves two purposes:
 provides a mechanism to describe a profile in an unambiguous way.
 This mechanism can be used to define additional profiles.
 
-- In addition it defines a **Core Profile** of the Thing Description 
-for use with selected protocols. The <a>Core Profile</a> formalizes
+- In addition it defines a **Base Profile** of the Thing Description
```
Comment (benfrancis) [inline README.md:17]: There is no "Base Profile". There is an "HTTP Baseline Profile" which may be renamed to "HTTP Basic Profile" (#265), but it only uses HTTP (not "selected profiles" as written here).

**Review by benfrancis — CHANGES_REQUESTED** [review]
The README changes seem unrelated to the WebHooks change, should that not be a separate PR?

**Comment by mlagally** [comment]
Profile call on Aug 31.:
Agree to merge after removal of README.md changes

**Pull request merged**